src/components/BaseButton.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <button v-bind="$attrs" @click="handleClick" class="button" :class="variant">
+    {{ label }}
+  </button>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from "vue";
+
+export default defineComponent({
```

Comment:
Add name to the components, to make it easier to find them in the Vue inspector
[components name](https://v2.vuejs.org/v2/style-guide/?redirect=true#Multi-word-component-names-essential)